google-api-client/src/main/java/com/google/api/client/googleapis/services/AbstractGoogleClient.java

```diff
@@ -232,6 +232,7 @@ public final BatchRequest batch() {
    * @return newly created Batch request
    */
   public final BatchRequest batch(HttpRequestInitializer httpRequestInitializer) {
+    @SuppressWarnings("deprecated")
     BatchRequest batch =
```

Comment:
Should we migrate this to the new API?

Reply:
This one is the method that users should be using that sets the batch path.